packages/jsii/bin/jsii.ts

```diff
@@ -54,6 +54,12 @@ const warningTypes = Object.keys(enabledWarnings);
             desc: `List of warnings to silence (warnings: ${warningTypes.join(
               ',',
             )})`,
+          })
+          .option('strip-deprecated', {
```

Comment:
Is there a way to ensure that when 'new' jsii features are supported (say, union types), they are correctly handled by this flag? 

Not expecting it in this PR, but wondering if you've thought about this. 

One way that comes to mind is - "some" automated mechanism (hand wave) of taking the jsii fixture (calc) and marking most or all of them as deprecated and stripping them out.

Another option - if there's some place where we've centralized the jsii specification and supported features, it might be easier walk through them and dynamically generate a fixture and run them through strip-deprecated.

Reply:
(Hand wave) Contributions are welcome 🤓